### PR TITLE
fixes kustomize build looking for argocd namespace by adding documentation to switch to project in the README

### PR DESCRIPTION
**Pull request by chadhellyea**
added ssh keys to gitignore and added oc project to make sure we were executing the kustomize command to the correct project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
 local/
```
Comment (sabre1041) [inline .gitignore:1]: This should be addressed here:

https://github.com/redhat-cop/rhel-edge-automation-arch/blob/main/openshift/gitops/clusters/overlays/byo/bootstrap/.gitignore

File: README.md [hunk 2]
```diff
@@ -76,6 +76,7 @@ Next, modify `openshift/gitops/clusters/overlays/byo/bootstrap/redhat-portal-cre
 We are now ready to bootstrap the environment. To do this, run:
 
 ```shell
+oc project openshift-gitops
 kustomize build --load-restrictor=LoadRestrictionsNone openshift/gitops/clusters/overlays/byo/bootstrap/ | oc apply -f -
```
Comment (sabre1041) [inline README.md:80]: Can we change `load-restrictor` to `load_restrictor` as there is a misspelling

File: README.md [hunk 3]
```diff
@@ -76,6 +76,7 @@ Next, modify `openshift/gitops/clusters/overlays/byo/bootstrap/redhat-portal-cre
 We are now ready to bootstrap the environment. To do this, run:
 
 ```shell
+oc project openshift-gitops
```
Comment (sabre1041) [inline README.md:79]: Do we want to fix the manifests instead to specify the namespace in the associated `kustomization.yaml` files

**Review by sabre1041 — COMMENTED** [review]
@chadhellyea A few comments for your thoughts

File: .gitignore (hunk 1 above)
Comment (chadhellyea) [reply .gitignore:1]: it was not ignoring from that location, and i accidentally pushed up my token was why i but it at the root level.

File: README.md (hunk 2 above)
Comment (chadhellyea) [reply README.md:80]: just tested it is correct as it is:
```shell
$ kustomize build --load_restrictor=LoadRestrictionsNone openshift/gitops/clusters/overlays/byo/bootstrap/ | oc apply -f -
Error: unknown flag: --load_restrictor

$ rhel-edge-automation-arch]$ kustomize build --load-restrictor=LoadRestrictionsNone openshift/gitops/clusters/overlays/byo/bootstrap/ | oc apply -f -
namespace/openshift-cnv created
namespace/rfe created
namespace/sealed-secrets created
secret/image-builder-ssh-key created
secret/redhat-portal-credentials created
appproject.argoproj.io/bootstrap created
appproject.argoproj.io/cluster-config created
appproject.argoproj.io/rfe created
```

**Comment by chadhellyea** [comment]
On a clean environment the issues were resolved. If you try and deploy to an environment from previous commits you will get this error. Recommend making sure you have a clean environment. Updated README to make the process cleaner.

**Pull request closed without merging**